deps/chakrashim/src/jsrtisolateshim.cc

```diff
@@ -377,4 +380,26 @@ void* IsolateShim::GetData(uint32_t slot) {
   return slot < _countof(this->embeddedData) ? embeddedData[slot] : nullptr;
 }
 
+JsValueRef IsolateShim::GetChakraShimJsArrayBuffer() {
+  if (this->chakraShimArrayBuffer == nullptr) {
+    CHAKRA_VERIFY(JsCreateExternalArrayBuffer(
+                  (void*)jsrt::chakra_shim_native,
+                  sizeof(jsrt::chakra_shim_native),
+                  nullptr, nullptr,
+                  &this->chakraShimArrayBuffer) == JsNoError);
```

Comment:
Not sure if these 2 jslib sources are needed multiple times. They are mostly only needed at initialization. No need to save/cache them in IsolateShim (also their lifetime may be a bit problematic, no explicit code to keep these cached Vars alive).